WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardPromptsCardCell.swift

```diff
@@ -554,20 +564,27 @@ private extension DashboardPromptsCardCell {
         var toAction: UIAction {
             switch self {
             case .viewMore(let handler),
-                    .skip(let handler),
-                    .remove(let handler):
-                return .init(title: title, image: image, attributes: menuAttributes, handler: { _ in
+                 .skip(let handler),
+                 .remove(let handler),
+                 .learnMore(let handler):
+                return UIAction(title: title, image: image, attributes: menuAttributes) { _ in
```

Comment:
I think we could just pass in the `handler` as an argument rather than creating a closure.

Reply:
The signature of the menu handler is `() -> Void` and this initializer takes a `(UIAction) -> Void` handler. I considered updating the menu's handler to match but this is also used by `MenuSheetViewController.MenuItem`.